### PR TITLE
Add Issue762 testcases

### PR DESCRIPTION
**Pull request by maryvictol**
This PR adds testcases for Billing quotas (https://github.com/epam/cloud-pipeline/issues/762)

### CODE REVIEW

File: docs/testcases/UI/Billing/762/762_2.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+| Steps | Actions | Expected results |
+|:---:| --- | --- |
+| 1 | Login as admin user from the prerequisites | |
+| 2 | Open the **Billing** page | |
+| 3 | Click **Quotas** tab | |
+| 4 | Click **Add quota** button | ***Create global quota*** modal opens |
+| 5 | Specify follow values for quota <li> *Quota* - `<quota2>` is more than `<billing1>` from the [762_1](762_1.md) case Prerequisites <li> quota period - by default <li> *Threshold* - `80` <li> *Action* - `Notify` | |
+| 6 | Click **Add Action** button and Specify follow values for Action: <li>*Threshold* - `110` <li> *Action* - `Read-only mode` and `Notify` | |
+| 7 | Input name of the non-admin user from the prerequisites into the **Recipients** field | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Billing/762/762_2.md]: Specify non-admin user into the current case prerequisites or fix to `Input name of the non-admin user from the prerequisites of the 762_1 case...`

File: docs/testcases/UI/Billing/762/762_2.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+| 8 | Click **Save** button | Error message `Overall quota per month already exists` appears |
+| 9 | Change ***Quota period*** value to `per quarter` | |
+| 10 | Click **Save** button | The following info appears in the ***Overall*** section: <li> `<quota>$ per quarter`, where `<quota>` is value specified at step 5 <li> `80%: notify 110% read-only mode` - pairs of threshold-action configured at steps 5-6 |
```
Comment (NShaforostov) [inline docs/testcases/UI/Billing/762/762_2.md]: > `80%: notify 110% read-only mode` - pairs of threshold-action...

fix to:
`80%: notify` and `110% read-only mode, notify` - pairs of threshold-action...

File: docs/testcases/UI/Billing/762/762_2.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+| 11 | Wait time equals to `<billing_quotas_period>` from the [762_1](762_1.md) case Preparations | | 
+| 12 | Open the **Settings** page | |
+| 13 | Click the **USER MANAGEMENT** tab | |
+| 14 | Click the **Users** tab | |
+| 15 | Find the non-admin user from the prerequisites | User has warning `Billing quotas exceeded` |
```
Comment (NShaforostov) [inline docs/testcases/UI/Billing/762/762_2.md]: Specify non-admin user into the current case prerequisites or fix to `Find the non-admin user from the prerequisites of the 762_1 case...`

File: docs/testcases/UI/Billing/762/762_2.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+| 16 | Hover over the warning | Tooltip that contains info <br> `Global monthly expenses <...>$,quota <quota1>$. Actions:90% Notify` |
+| 17 | Find the admin user from the prerequisites | User hasn't warning `Billing quotas exceeded` |
+| 18 | Open the **Billing** page and click **Quotas** tab | <li> Circle isn't shown near the quota created at step 10 <li> Yellow circle is shown near the quotas created at step 20 of the [762_1](762_1.md) case |
+| 19 | Click *Remove* icon near the quota created at step 20 of [762_1](762_1.md) case | Confirmation pop-up appears |
+| 20 | Click **OK** button | Quota disappears from the quotas list |
+| 21 | Click on the quota created at step 12 | ***Global quota*** modal opens |
```
Comment (NShaforostov) [inline docs/testcases/UI/Billing/762/762_2.md]: Fix to `step 10`

File: docs/testcases/UI/Billing/762/762_7.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as admin user from the prerequisites | |
+| 2 | Open the **Billing** page | |
+| 3 | Click **Storages** tab in the **Quotas** section |  |
+| 4 | Click **Add quota** button for ***Billing centers*** section | ***Create storages quota*** modal opens that contains: <li> ***Billing center*** field <li> ***Quota*** field, <li> ***Quota period*** dropdown list with values `per month`, `per quarter`, `per year` (default value is `per month`) <li> ***Actions*** section that contains ***Threshold*** and ***Action*** fields and **Add Action** button <li> **Cancel** and **Save** buttons. |
+| 5 | Click into the ***Billing center*** field and select `<Billing_center>` from the prerequisites | | 
+| 6 | Specify follow values for quota <li> *Quota* - `<billing1>` <li> quota period -`per quarter` <li> *Threshold* - `70` <li> *Action* - `Read-only mode` | |
+| 7 | Click **Save** button | The following info appears in the ***Overall*** section: <li> `<Billing_center>: <quota>$ per quarter`, where `<quota>` is value specified at step 5 <li> `70%: read-only mode` - pair of threshold-action configured at step 7-9 |
```
Comment (NShaforostov) [inline docs/testcases/UI/Billing/762/762_7.md]: Fix `step 7-9` to `step 6`

File: docs/testcases/UI/Billing/762/762_7.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+| 8 | Wait time equals to `<billing_quotas_period>` from the [762_1](762_1.md) case Preparations and refresh page | Yellow circle is shown near the quota created at step 7|
+| 9 | Open the **Library** page | |
+| 10 | Open the `storage1` created at step 2 of Preparations | Storage page opens and is available for editing |
+| 11 | Create `file1.txt` | | 
+| 12 | Open the `fs_storage` created at step 2 of Preparations | Storage page opens and is available for editing |
+| 13 | Logout | |
+| 14 | Login as non-admin user from the prerequisites | |
+| 15 | Repeat steps 10-11, 13 | For both cases Storage page opens and doesn't contain <li> **Create** and **Upload** buttons for whole table <li> **Edit** and **Remove** buttons for each row |
```
Comment (NShaforostov) [inline docs/testcases/UI/Billing/762/762_7.md]: Fix `steps 10-11, 13` to `steps 9-10, 12`

File: docs/testcases/UI/Billing/762/762_5.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+| 24 | Execute command: <br> `pipe run -di <test_tool> -u <non-admin_user>`, <br> where `<test_tool>` is the test tool name | Response `Error: Failed to fetch data from server. Server responded with message: Launch of new compute instances is forbidden due to exceeded billing quota.` is returned |
+| 25 | Open the **Settings** page | |
+| 26 | Click the **USER MANAGEMENT** tab | |
+| 27 | Click the **Users** tab | |
+| 28 | Find the non-admin user from the prerequisites | User has warning `Billing quotas exceeded` |
+| 29 | Hover over the warning | Tooltip that contains info <br> `Billing center <Billing_center>: compute quarterly expenses <...>$,quota <billing1>$. Actions:70% Disable new jobs` <br> `User <non_admin_user>: compute annual expenses <...>$,quota <billing2>$. Actions:70% Stop all jobs` |
+| 30 | Find the admin user from the prerequisites | User hasn't warning `Billing quotas exceeded` |
+
+**After:**
+- Remove quotas created at steps 6, 10
```
Comment (NShaforostov) [inline docs/testcases/UI/Billing/762/762_5.md:61]: Add rows:
- Terminate the run paused at step 7 of the case preparations
- Stop the run launched at step 9 of the case preparations

File: docs/testcases/UI/Billing/762/762_7.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+| 16 | At the **Runs** page, click the run launched at step 6 of the Preparations | |
+| 17 | Wait until the **SSH** hyperlink appears | |
+| 18 | Click the **SSH** hyperlink | |
+| 19 | Execute command: <br> `echo test >> /cloud-data/<storage2_name with path>/test_file1.txt`, <br>where `<storage2_name with path>` is the full path to the `storage2` created at step 2 of the Preparations |  <li>`test_file1.txt` file isn't created <li> Response `Read-only file system` is shown |
+| 20 | Execute command: <br> `pipe storage cp cp://<storage1>/file1.txt cp://<storage2>/`, <br> where `<storage1>` and `<storage2>` are the storages created at step 2 of the Preparations |  <li>`file1.txt` file isn't copied <li> Response `Read-only file system` is shown |
+| 21 | Execute command: <br> `echo test >> /cloud-data/<fs_storage>/test_file1.txt`, <br> where `<fs_storage>` is the full path to the `fs_storage` created at step 2 of the Preparations |  <li>`test_file1.txt` file isn't created <li> Response `Read-only file system` is shown |
+
+**After:**
+- Remove quota created at step 7
```
Comment (NShaforostov) [inline docs/testcases/UI/Billing/762/762_7.md:47]: Add row:
- Stop the run launched at step 6 of the case preparations

File: docs/testcases/UI/Billing/762/762_8.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+    - *Storages* billing for the admin user from the Prerequisites for the current year is `<billing2>`
+- Perform [762_1](762_1.md) case
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as admin user from the prerequisites | |
+| 2 | Open the **Billing** page | |
+| 3 | Click **Storages** tab in the **Quotas** section | |
+| 4 | Select **Users** section | | 
+| 5 | Click **Add quota** button for ***Billing centers*** section | ***Create storages quota*** modal opens that contains: <li> ***User*** field <li> ***Quota*** field, <li> ***Quota period*** dropdown list with values `per month`, `per quarter`, `per year` (default value is `per month`) <li> ***Actions*** section that contains ***Threshold*** and ***Action*** fields and **Add Action** button <li> **Cancel** and **Save** buttons. |
```
Comment (NShaforostov) [inline docs/testcases/UI/Billing/762/762_8.md]: > Click **Add quota** button for ***Billing centers*** section

Fix to "...for **Users** section"

File: docs/testcases/UI/Billing/762/762_8.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+| 12 | Click the **USER MANAGEMENT** tab | |
+| 13 | Click the **Users** tab | |
+| 14 | Find the non-admin user from the prerequisites | User has warning `Billing quotas exceeded` |
+| 15 | Hover over the warning | Tooltip that contains info <br> `User <user_name>: storages annual expenses <...>$,quota <billing1>$. Actions:70% Block`, where `<user_name>` is non-admin user from Prerequisites |
+| 16 | Logout | |
+| 17 | Try to login as non-admin user from the prerequisites | The page with the authorization error appears |
+| 18 | Login as admin user from the prerequisites | |
+| 19 | Repeat steps 2-4 | |
+| 20 | Delete quota created at step 8 | |
+| 21 | Repeat steps 17-18 | User is logged in successfully |
```
Comment (NShaforostov) [inline docs/testcases/UI/Billing/762/762_8.md]: Fix `steps 17-18` to `steps 16-17`

**Review by NShaforostov — APPROVED** [review]
Before the merge, fix minor notes according to the comments, please.

File: docs/testcases/UI/Billing/762/762_2.md (hunk 1 above)
Comment (maryvictol) [reply docs/testcases/UI/Billing/762/762_2.md]: Done

File: docs/testcases/UI/Billing/762/762_8.md (hunk 10 above)
Comment (maryvictol) [reply docs/testcases/UI/Billing/762/762_8.md]: Done

File: docs/testcases/UI/Billing/762/762_8.md (hunk 9 above)
Comment (maryvictol) [reply docs/testcases/UI/Billing/762/762_8.md]: Done

File: docs/testcases/UI/Billing/762/762_7.md (hunk 8 above)
Comment (maryvictol) [reply docs/testcases/UI/Billing/762/762_7.md:47]: Done

File: docs/testcases/UI/Billing/762/762_5.md (hunk 7 above)
Comment (maryvictol) [reply docs/testcases/UI/Billing/762/762_5.md:61]: Done

File: docs/testcases/UI/Billing/762/762_7.md (hunk 6 above)
Comment (maryvictol) [reply docs/testcases/UI/Billing/762/762_7.md]: Done

File: docs/testcases/UI/Billing/762/762_7.md (hunk 5 above)
Comment (maryvictol) [reply docs/testcases/UI/Billing/762/762_7.md]: Done

File: docs/testcases/UI/Billing/762/762_2.md (hunk 4 above)
Comment (maryvictol) [reply docs/testcases/UI/Billing/762/762_2.md]: Done

File: docs/testcases/UI/Billing/762/762_2.md (hunk 2 above)
Comment (maryvictol) [reply docs/testcases/UI/Billing/762/762_2.md]: Done

File: docs/testcases/UI/Billing/762/762_2.md (hunk 3 above)
Comment (maryvictol) [reply docs/testcases/UI/Billing/762/762_2.md]: Done

**Pull request merged**